libs/elements/cards/src/lib/material.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from "@angular/core";
+import { MatIconModule } from '@angular/material/icon'
+
```

Comment:
This shouldn't be here. But I'll set this up on the project. Check for updates in a few

Reply:
Anytime